crypto.go

```diff
@@ -101,8 +99,8 @@ func (i *Identity) ConfigForPeer(
 	return conf, keyCh
 }
 
-// getRemotePubKey derives the remote's public key from the certificate chain.
-func getRemotePubKey(chain []*x509.Certificate) (ic.PubKey, error) {
+// PubKeyFromCertChain derives the remote's public key from the certificate chain.
```

Comment:
Documentation nit: it _extracts_ the public key and _verifies_ the chain.

Reply:
Fixed.